client/reducers/products.js~

```diff
@@ -0,0 +1,38 @@
+import axios from 'axios';
+
+//Actions Types
+const GET_PRODUCTS = 'GET_PRODUCTS';
+
+//Action Creators
+export function getProducts(products) {
+	type: GET_PRODUCTS,
+	products
+};
+
+//Thunk Creators
+export function fetchProducts() {
+	console.log('calling fetchproducts')
+	return function thunk(dispatch) {
+		axios.get(`/api/products`)
+			.then((products)=> {
+				console.log('success get')
+				dispatch(getProducts(products.data))
+			})
+	}
+}
+
+//Action Reducer
+const productReducer = function(state=[], action) {
+	switch(action.type) {
+<<<<<<< HEAD
```

Comment:
You have merge conflict lines here. 
the <<<<<<<<< HEAD, and other lines. Need to delete those lines if you've fixed the conflict already.